README.md

```diff
@@ -68,17 +68,18 @@ Or, you can add Lift to your `pom.xml` like so:
 
     <dependency>
       <groupId>net.liftweb</groupId>
-      <artifactId>lift-mapper_${scala.version}</artifactId>
-      <version>2.5.1</version>
+      <artifactId>lift-webkit_${scala.version}</artifactId>
+      <version>3.0.1</version>
```

Comment:
Given that we don't have any reports of serious issues, we could go ahead and just set this to `3.1.0` (here and in the above change as well) and then merge it when the 3.1.0 release goes out.